frontend/src/app/organization/organization.service.ts

```diff
@@ -19,17 +19,22 @@ export class OrganizationService {
   private organizationEvent = new BehaviorSubject<Organization | null>(null);
   private organizationsEvent = new Subject<Organization[]>();
   private offersEvent = new Subject<Offer[]>();
+  private createOrganizationEvent = new Subject<CreateOrEditOrganization>();
 
   public contactStatus$: Observable<ContactStatus> = this.contactStatusEvent.asObservable();
   public organization$: Observable<Organization | null> = this.organizationEvent.asObservable();
   public organizations$: Observable<Organization[]> = this.organizationsEvent.asObservable();
   public offers$: Observable<Offer[]> = this.offersEvent.asObservable();
+  public createOrEditOrganization$: Observable<CreateOrEditOrganization> = this.createOrganizationEvent.asObservable();
```

Comment:
I'm not a fan of this solution with `createOrEditOrganization$` - I really don't know if that's a industry standard here or not but from my perspective it looks really bizare.

Reply:
@magul Although the current name of the event was my proposition, since the previous one was misleading as for the purpose of this event, I can't tell if the pattern itself here is good or not for that particular situation. 

We can change the naming so it looks better, but still this will be the event that handles both 'create' and 'edit' events. 

Do we have anyone on the team with broad experience of RxJS patterns to take a look at this?